scripts/toss.coffee

```diff
@@ -15,7 +15,7 @@
 
 
 module.exports = (robot) ->
-  robot.hear /toss/i, (res) ->
+  robot.respond /toss/i, (res) ->
```

Comment:
That's not an excuse!

This is your work and not mangal's. You are responsible for the code.